src/messages.rs

```diff
@@ -36,13 +36,11 @@ use std::collections::{BTreeMap, BTreeSet};
 use std::fmt::{self, Debug, Formatter};
 use std::iter;
 use std::time::Duration;
+use super::QUORUM;
 use types::MessageId;
 use utils;
 use xor_name::XorName;
 
-/// The quorum, as a percentage of the group size.
-const QUORUM: usize = 60;
```

Comment:
You're right, I'd also go with `>=`: That way `QUORUM = 100` still make sense, and I think `51` is also intuitive. (Not sure about what value we should actually choose: Everything between `51` and `100` is fine until we decide on a particular value based on the simulations.)